File: oeml-sdk/javascript-closure-angular/API/Client/CancelOrderSingleRequest.js
goog.provide('API.Client.CancelOrderSingleRequest');

/**
 * @record
 */
API.Client.CancelOrderSingleRequest = function() {}

/**
 * Exchange name
 * @type {!string}
 * @export
 */
API.Client.CancelOrderSingleRequest.prototype.exchangeId;

/**
 * Order Id
 * @type {!string}
 * @export
 */
API.Client.CancelOrderSingleRequest.prototype.exchangeOrderId;

/**
 * Client order Id
 * @type {!string}
 * @export
 */
API.Client.CancelOrderSingleRequest.prototype.clientOrderId;

File: oeml-sdk/dart-jaguar.yaml/lib/model/cancel_order_single_request.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'cancel_order_single_request.jser.dart';

class CancelOrderSingleRequest {
   /* Exchange name */
  @Alias('exchange_id', isNullable: false,  )
  final String exchangeId;
   /* Order Id */
  @Alias('exchange_order_id', isNullable: false,  )
  final String exchangeOrderId;
   /* Client order Id */
  @Alias('client_order_id', isNullable: false,  )
  final String clientOrderId;
  

  CancelOrderSingleRequest(
      

{
     this.exchangeId = null,  
     this.exchangeOrderId = null,  
     this.clientOrderId = null 
    
    }
  );

  @override
  String toString() {
    return 'CancelOrderSingleRequest[exchangeId=$exchangeId, exchangeOrderId=$exchangeOrderId, clientOrderId=$clientOrderId, ]';
  }
}

@GenSerializer(nullableFields: true)
class CancelOrderSingleRequestSerializer extends Serializer<CancelOrderSingleRequest> with _$CancelOrderSingleRequestSerializer {

}

